.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -15,10 +15,9 @@ assignees: ""
 Provide at least:
 
 - OS:
-- `pip list` of the host Python where `tox` is installed in the dropdown below
 
 <details open>
-<summary>Output of <code>pip list</code></summary>
+<summary>Output of <code>pip list</code> of the host Python, where <code>tox</code> is installed</summary>
```

Comment:
BTW, I made the dropdown default to "open" so it's backwards compatible (shows `pip list` and `tox -rvv` output by default) by using `<details open>`.

If you want, I can make it `<details>`, which defaults to closed.  Up to you